qa/tasks/vstart_runner.py

```diff
@@ -1247,6 +1259,8 @@ def exec_test():
         elif f == "--kclient":
             use_kernel_client = True
         elif f == '--usens':
+            if mon_in_localhost():
+                raise RuntimeError("Mon in localhost, '--usens' option is forbiddened!")
```

Comment:
Execution should not be terminated when MON is on localhost and  `--create` flag is passed since vstart_runner.py would create a new cluster without using localhost -

```
$ python3 ../qa/tasks/vstart_runner.py tasks.cephfs.test_cephfs_shell.TestCD.test_cd_with_args --usens --brxnet=172.16.0.0/16 --create
Using guessed paths /home/rishabh/repos/ceph/test2/build/lib/ ['/home/rishabh/repos/ceph/test2/qa', '/home/rishabh/repos/ceph/test2/build/lib/cython_modules/lib.3', '/home/rishabh/repos/ceph/test2/src/pybind']
Using guessed paths /home/rishabh/repos/ceph/test2/build/lib/ ['/home/rishabh/repos/ceph/test2/qa', '/home/rishabh/repos/ceph/test2/build/lib/cython_modules/lib.3', '/home/rishabh/repos/ceph/test2/src/pybind']
2020-06-11 16:27:39,609.609 INFO:__main__:Mon in localhost and will disable the netns unsharing for mountpoint!
Traceback (most recent call last):
  File "../qa/tasks/vstart_runner.py", line 1505, in <module>
    exec_test()
  File "../qa/tasks/vstart_runner.py", line 1262, in exec_test
    raise RuntimeError("Mon in localhost, '--usens' option is forbiddened!")
RuntimeError: Mon in localhost, '--usens' option is forbiddened!
```

Reply:
It would also be nice to tell the user in the message that either the user can get rid of `--usens` or pass `--create`.

Reply:
Sure, I will fix it.
Thanks.